src/actor.rs

```diff
@@ -3,8 +3,8 @@ use crate::traits::{LoadableState, UptimeCheckerActor};
 use crate::types::{InitParams, MultiAddr, NodeInfo, NodeInfoPayload, PeerID, ReportPayload};
 use crate::{ensure, Error};
 
-const THRESHOLD_NUM: usize = 20000;
-const THRESHOLD_DUM: usize = 30000;
+const THRESHOLD_NUMERATOR: usize = 20000;
```

Comment:
Why can't this be directly a float?
```suggestion
const THRESHOLD = 2/3:
```

Reply:
It's just for convenience I think. First if 2/3 is represented as a float, there are some lost precision. Representing with Numerator + Denominator can be flexible. At the same time, when using float, I need to cast total votes to `f64`. For this use case, I think should be ok, but usize -> u64, not sure there will be underflow or anything. And once multiplied, I need to round down/round up and convert back to usize to compare with count. If I just use NUMBERATOR + DENOMINATOR, I dont have to perform the casting or rounding.

Reply:
OK, I am convinced :) Can you add a few lines in the comments explaining this?? Thanks!